### PR TITLE
Updated retention descriptions and added new retention fields

### PR DESCRIPTION
**Pull request by sri-byte**
Impact: Updated retention descriptions and added new 28 day and above retention fields

### CODE REVIEW

File: views/mattermost/server_fact.view.lkml [hunk 1]
```diff
@@ -82,100 +82,114 @@ view: server_fact {
...
-  dimension: retention_28day_flag {
-    label: " 28-Day Retention"
-    group_label: " Telemetry Flags"
-    description: "Boolean indicating the instance was retained after 28 days since their first active date. This metric is a flag indicating users performed events between hour 672 and 696 from the instance's first active timestamp."
-    type: yesno
-    sql: COALESCE(${TABLE}.retention_28day_flag, false) ;;
+  dimension: retention_0day_flag {
+    label: "0-Day Retention"
+    group_label: "Telemetry Flags"
+    description: "Boolean indicating the instance was retained between (instance) creation time and 1 day since their first active date."
```
Comment (esethna) [inline views/mattermost/server_fact.view.lkml:88]: Is this essentially saying that if there are ANY real time telemetry events that fire after server creation time (but before 1 day) then this is true?

File: views/mattermost/server_fact.view.lkml [hunk 2]
```diff
@@ -82,100 +82,114 @@ view: server_fact {
...
-  dimension: retention_1week_flag {
+  dimension: retention_28day_above_flag {
+    label: " 28-Day Retention"
+    group_label: " Telemetry Flags"
+    description: "Boolean indicating the instance was retained after 28 days and above since their first active date. "
+    sql: COALESCE(${TABLE}.retention_28day_above_flag, false) ;;
+  }
+
+  dimension: retention_28day_above_users {
+    label: " 28-Day Retained Users"
```
Comment (esethna) [inline views/mattermost/server_fact.view.lkml:177]: duplicate label

File: views/mattermost/server_fact.view.lkml (hunk 1 above)
Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:88]: we are excluding the ones generated by the instance such as `category = performance` as of now. There was another category of events that were triggered by the instance, we need to update those as well, I will need your help review these events.

Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:88]: last time I checked, there was one other one that gets triggered after 14 days of inactivity, called `inactive_server_email_sent`, a notification which goes via email, when clicked brings them back to the server. This was mostly seen in Cloud.

Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:88]: I have the link for the events that get triggered right after an instance is created, send me a message on MM.

**Review by esethna — APPROVED** [review]

**Comment by esethna** [comment]
Thanks!

**Pull request merged**